lib/article_card.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_web_browser/flutter_web_browser.dart';
+
+class _ArticleDescription extends StatelessWidget {
+  _ArticleDescription({
+    Key key,
+    this.title,
+    this.subtitle,
+    this.author,
+    this.publishDate,
+  }) : super(key: key);
+
+  final String title;
+  final String subtitle;
+  final String author;
+  final String publishDate;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        Expanded(
+          flex: 1,
+          child: Column(
```

Comment:
Is it neccessary for the Column in Column? Or could the first Column be Container?

Reply:
I think it can remain a column because we could add more features later on, such as reactions and comments. Whereas with a container, that would not be possible.

Reply:
If that's the case, why not Row in Column or Column in Row? Wouldn't that be better in avoiding overlapping?